cmake/FindQGIS.cmake

```diff
@@ -63,15 +70,15 @@ IF(WIN32)
         "$ENV{OSGEO4W_ROOT}/apps/${OSGEO4W_QGIS_SUBDIR}/lib"
         "$ENV{PROGRAMFILES}/QGIS/lib"
     )
-    # FIND_LIBRARY(QGIS_ANALYSIS_LIBRARY
-    #   NAMES qgis_analysis
-    #   PATHS
-    #     "$ENV{LIB_DIR}"
-    #     "$ENV{LIB}"
-    #     "$ENV{OSGEO4W_ROOT}/lib"
-    #     "$ENV{OSGEO4W_ROOT}/apps/${OSGEO4W_QGIS_SUBDIR}/lib"
-    #     "$ENV{PROGRAMFILES}/QGIS/lib"
-    # )
+    FIND_LIBRARY(QGIS_ANALYSIS_LIBRARY
```

Comment:
Is that why the :apple: fails now (I remember seeing @3nids commenting this out during his iOS integration)? Should it be addressed in this PR or separately?